src/learning-catalogue/model/dateRange.ts

```diff
@@ -27,5 +27,7 @@ export class DateRange {
 		groups: ['all', 'event.all', 'event.dateRanges.endTime'],
 		message: 'validation_module_event_dateRanges_endBeforeStart',
 	})
-	endTime: String
+	endTime: string
+
+	formattedDate: string
```

Comment:
If we add `formattedDate` here it will then become serialised in ElasticSearch when we store it in back-end - if it's about formatting date to something like `1 September 2018` can we use a filter `formatDate` (see `server.ts`) or if it's not feasible make it a method and call the method in the template?